LondonBritishLibrary/orient/BLorient803.xml

```diff
@@ -130,13 +89,134 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+<incipit xml:lang="gez">አንቀጽ፡ ተዋርሶ፡ ዘተሰነዓዉ፡ ላዕሌሁ፡ እግዚእነ፡ አባ፡ ቂርሎስ፡ ሊቀ፡ ጳጳሳት፡ ዘእለ፡ እስክንድርያ፡
+  ምስለ፡ ኤጲስ፡ ቆጶሳቲሁ፡ ወሊቃናቲሁ፡ ወመኳንንቲሁ፡ ወውእቱ፡ ፷ምዕራፍ፡ ወመብዝኅቱኒ፡ እምነ፡ ዘጸሐፎ፡ አባ፡ ቆዝምስ፡ ፩እምሊቃነ፡
+  ጳጳሳት፡ ዘእለ፡ እስክንድርያ። ወይቤ፡ በተፍጻሜቱ፡ ከመ፡ ውእቱ፡ እምዘ፡ ሠርዕዎ፡ እምሊቃነ፡ ሕግ፡ ወተረፈ፡ ዚአሁ፡ እምልብ፡ ወዘልማድ፡
+  ወዝንቱ፡ ተሠርዓ፡ አመ፡ ዓሡሩ፡ ወሰቡዑ፡ ለወርኃ፡ መስከረም፡
+  በ<date calendar="diocletian" when="0958">፱፻፶ወ፰ዓመተ፡ ሰማዕታት፡</date> ንጹሓን፡ ትንብልናሆሙ፡ ተሀሉ፡ ምስሌነ፡ አሜን። </incipit>
+<textLang mainLang="gez"/>
+</msItem>
+
+<colophon xml:id="coloph1" xml:lang="gez">
+<locus target="#147v"></locus>
+መልአ፡ ጸሒፈ፡ መጽሐፈ፡ ቀኖና፡ ዘፍትሐ፡ ኵነኔ፡ ሕግ፡ መሲሓዊት፡ በዕለተ፡ ረቡዕ፡ አመ፡ ፩ለጥቅምት፡
+<date calendar="world" when="7172">በ፸፻ወ፩፻፸ወ፪ዓመተ፡ ዓለም፤</date> ወተፈጸመ፡ በዘመነ፡ ማርቆስ፡ ወንጌላዊ፡ በ፳አብቅቴ፡ በ፳ወ፪ሠርቀ፡ ሌሊት፡ ወ፩ሠርቀ፡
+መዓልተ:<gap reason="omitted" resp="PRS10171Wright"/> ወበ፲አርባዕቱ፡ ዓመተ፡ መንግሥቱ፡
+ለንጉሠ፡ ሃይማኖት፡ ዮሐንስ፡ ወለእግዚአብሔር፡ አኰቴት፡ እስከ፡ ለዓለም። ወዘአዕለዋ፡ ኅፁፅ፡ ወኃጥእ <gap reason="omitted" resp="PRS10171Wright"/>
+ወዘአጽሐፋኒ፡<gap reason="omitted" resp="PRS10171Wright"/>ወልደ፡ አቡነ፡ ትክለ ሃይማኖት፡
+<note>The colophon states that the manuscript was written in the 14th year of the reign of
+king <persName ref="PRS10375Yohanne">Yoḥannǝs</persName> i. e. 1681.</note>
+<note>The colophon has been altered by various erasures and corrections, from the words
+<foreign xml:lang="gez">በ፸፻ወ፩፻፸ወ፪</foreign> (originally <foreign xml:lang="gez">ወ፬) ዓመተ፡ ዓለም፡</foreign> to the
+ words <foreign xml:lang="gez">ወ፬ሠርቀ፡ መዓልት፡</foreign> after which three lines have been erased.
+The rest has been left untouched, with the exception of the name of the owner, which
+has disappeared on <locus target="#148r">f. 148r</locus>, as well as on <locus target="#2r">f. 2r</locus>.</note>
+</colophon>
+</msItem>
+</msContents>
+
+<physDesc>
+<handDesc>
+      <handNote script="Ethiopic" xml:id="h1">
+          <desc>The manuscript is well written in a good hand, and dated <date when="1681"/>,
+          </desc>
+          <date when="1681"/>
+      </handNote>
+  </handDesc>
+  <additions>
+      <list>
+
+<item xml:id="a1">
+<locus target="#1r"/>
+<desc type="OwnershipNote">The note stating that the manuscript belonged
+to king <persName ref="PRS5616IyasuI">ʾIyāsu</persName>, successor of <persName ref="PRS10375Yohanne">Yoḥannǝs</persName>.</desc>
+<q xml:lang="gez">ዘንጉሠ፡ ነገሥት፡ ኢያሱ።</q>
+</item>
+
+  <item xml:id="a2">
+  <locus target="#1r"/>
+  <desc type="OwnershipNote">The note states that the manuscript belongs to the church of
+  <placeName ref="INS0101MadhaneAlam"/>.</desc>
+  <q xml:lang="gez">ፍትሐ፡ ነገሥት፡ ዘ<placeName ref="INS0101MadhaneAlam">ቅዱስ፡ መድኃኔ፡ ዓለም።</placeName>
+     </q>
+      </item>
+      </list>
+  </additions>
+
+</physDesc>
+
+</msPart>
+
+<msPart xml:id="p2">
+<msIdentifier></msIdentifier>
+<msContents>
+<msItem xml:id="p2_i1">
+<locus target="#149r"/>
+<title type="incomplete" ref="LIT1395Fethan"/>
+<note>F. 149r is a leaf from another, unfinished manuscript of Fǝtḥa nagaśt, corrsponding to
```

Comment:
It is always good to give the folio number with locus (without writing it in the content), if there is no need for a special rendering of it. This will contribute to higher homogeneity on the app.